src/commands/build/wranglerjs/output.rs

```diff
@@ -6,7 +6,7 @@ use serde::Deserialize;
 use std::io::prelude::*;
 
 // This structure represents the communication between {wranglerjs} and
-// {wrangler}. It is send back after {wranglerjs} completion.
+// {wrangler}. It is sent back after {wranglerjs} completion.
 // FIXME(sven): make this private
```

Comment:
i guess while we're at it
```suggestion
// TODO: (sven) make this private
```